settings/src/test/java/fr/pilato/elasticsearch/crawler/fs/settings/FsSettingsParserTest.java

```diff
@@ -53,6 +54,7 @@
             .setUpdateRate(TimeValue.timeValueMinutes(5))
             .setIndexContent(true)
             .setOcr(OCR_FULL)
+            .setTikaCustomParsers(new ArrayList<>())
```

Comment:
I think you should add at least a singleton List of one `CustomTikaParser` element here to make sure serialisation and deserialisation is working well.